<ui version="4.0" >
 <class>DockFindInFiles</class>
 <widget class="QDockWidget" name="FindInFiles" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Find in files</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="labText" >
         <property name="text" >
          <string>Text to search for :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbText" >
         <property name="editable" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Options</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QCheckBox" name="chBoxWhole" >
            <property name="text" >
             <string>Whole word</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chBoxMatch" >
            <property name="text" >
             <string>Match case</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="chBoxReg" >
            <property name="text" >
             <string>Regular expression</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>Scope</string>
         </property>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>9</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rbPro" >
            <property name="text" >
             <string>Project files</string>
            </property>
            <property name="checked" >
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbOpen" >
            <property name="text" >
             <string>Open files</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rbSearch" >
            <property name="text" >
             <string>Search path</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="gbSearch" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="title" >
        <string>Search path</string>
       </property>
       <layout class="QGridLayout" >
        <property name="margin" >
         <number>9</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item row="1" column="1" >
         <widget class="QLineEdit" name="leMask" >
          <property name="text" >
           <string>*.h *.cpp</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QLabel" name="labMask" >
          <property name="text" >
           <string>Mask :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" >
         <widget class="QLineEdit" name="lePath" >
          <property name="palette" >
           <palette>
            <active>
             <colorrole role="WindowText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>207</red>
                <green>207</green>
                <blue>207</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Light" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Midlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>231</red>
                <green>231</green>
                <blue>231</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Dark" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>103</red>
                <green>103</green>
                <blue>103</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Mid" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>138</red>
                <green>138</green>
                <blue>138</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="BrightText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>228</red>
                <green>248</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>207</red>
                <green>207</green>
                <blue>207</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Shadow" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>127</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="HighlightedText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Link" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="LinkVisited" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>231</red>
                <green>231</green>
                <blue>231</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="WindowText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>207</red>
                <green>207</green>
                <blue>207</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Light" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Midlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>238</red>
                <green>238</green>
                <blue>238</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Dark" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>103</red>
                <green>103</green>
                <blue>103</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Mid" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>138</red>
                <green>138</green>
                <blue>138</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="BrightText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>228</red>
                <green>248</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>207</red>
                <green>207</green>
                <blue>207</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Shadow" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>127</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="HighlightedText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Link" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="LinkVisited" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>231</red>
                <green>231</green>
                <blue>231</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="WindowText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Button" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>207</red>
                <green>207</green>
                <blue>207</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Light" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Midlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>238</red>
                <green>238</green>
                <blue>238</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Dark" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>103</red>
                <green>103</green>
                <blue>103</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Mid" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>138</red>
                <green>138</green>
                <blue>138</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Text" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="BrightText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="ButtonText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Base" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>255</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Window" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>207</red>
                <green>207</green>
                <blue>207</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Shadow" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>0</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>127</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="HighlightedText" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>128</red>
                <green>128</green>
                <blue>128</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Link" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>0</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="LinkVisited" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>255</red>
                <green>0</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="AlternateBase" >
              <brush brushstyle="SolidPattern" >
               <color alpha="255" >
                <red>231</red>
                <green>231</green>
                <blue>231</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
         </widget>
        </item>
        <item row="0" column="0" >
         <widget class="QLabel" name="labPath" >
          <property name="text" >
           <string>Path :</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" >
         <widget class="QToolButton" name="tbPath" >
          <property name="text" >
           <string>...</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2" >
         <widget class="QCheckBox" name="chBoxRec" >
          <property name="text" >
           <string>Recurse subdirectories</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>131</width>
           <height>31</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbOk" >
         <property name="text" >
          <string>Find</string>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbClose" >
         <property name="text" >
          <string>Close</string>
         </property>
         <property name="autoDefault" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbText</tabstop>
  <tabstop>chBoxWhole</tabstop>
  <tabstop>chBoxMatch</tabstop>
  <tabstop>chBoxReg</tabstop>
  <tabstop>rbPro</tabstop>
  <tabstop>rbOpen</tabstop>
  <tabstop>rbSearch</tabstop>
  <tabstop>lePath</tabstop>
  <tabstop>tbPath</tabstop>
  <tabstop>leMask</tabstop>
  <tabstop>chBoxRec</tabstop>
  <tabstop>pbOk</tabstop>
  <tabstop>pbClose</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>pbOk</sender>
   <signal>clicked()</signal>
   <receiver>FindInFiles</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>209</x>
     <y>334</y>
    </hint>
    <hint type="destinationlabel" >
     <x>96</x>
     <y>254</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rbSearch</sender>
   <signal>toggled(bool)</signal>
   <receiver>gbSearch</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>207</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel" >
     <x>168</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>FindInFiles</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>264</x>
     <y>330</y>
    </hint>
    <hint type="destinationlabel" >
     <x>69</x>
     <y>326</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
